browser/ui/webui/playlist_ui.cc

```diff
@@ -25,6 +25,43 @@
 
 namespace playlist {
 
+namespace {
```

Comment:
Nit:  I found `#include "chrome/browser/profiles/profile.h"` in the above license sentence.
It would be good to delete while you're touching this file :)

Reply:
Oh, well spotted! Removed.